### DISCUSSION TITLE
Gauge plugin and bar plugin in open MCT.

### DISCUSSION THREAD

**Discussion by Vaibhav2121** [nasa/openmct#3807] [Community Q&A] [answered]
is gauge plugin and bar gauge plugin available in open MCT.

**Comment by DusterTheFirst** [comment]
This may be better suited as a discussion topic: https://github.com/nasa/openmct/discussions/categories/community-q-a

**Comment by jvigliotta** [comment]
@Vaibhav2121 @danielpacak moving this over to discussions as well.

**Comment by davetsay** [comment]
@davetsay

**Comment by davetsay** [comment, marked as answer]
@Vaibhav2121 , they are! The Gauge plugin is installed by default. Gauges are creatable objects and the available types are listed below.
- Filled Dial
- Needle Dial
- Vertical Meter
- Meter Inverted
- Horizontal Meter
